doc/classes/CollisionPolygon2D.xml

```diff
@@ -19,6 +19,10 @@
 		<member name="one_way_collision" type="bool" setter="set_one_way_collision" getter="is_one_way_collision_enabled" default="false">
 			If [code]true[/code], only edges that face up, relative to [CollisionPolygon2D]'s rotation, will collide with other objects.
 			[b]Note:[/b] This property has no effect if this [CollisionPolygon2D] is a child of an [Area2D] node.
+			[b]Note:[/b] The one way collision direction can be configured by setting [member one_way_collision_direction].
```

Comment:
The first and last lines sound like they contradict each other now. (_"relative to [CollisionPolygon]'s rotation_ VS. _"The one way collision direction can be configured"_). I have no suggestions as of writing this.